src/main/java/edu/greenblitz/tobyDetermined/subsystems/DriveTrain/IO/GyroIOs/Gyro.java

```diff
@@ -0,0 +1,53 @@
+package edu.greenblitz.tobyDetermined.subsystems.DriveTrain.IO.GyroIOs;
+
+import edu.greenblitz.tobyDetermined.RobotMap;
+
+public class Gyro {
+
+    private GyroIO io;
+    private final GyroIOInputsAutoLogged inputs = new GyroIOInputsAutoLogged();
+
+    public Gyro() {
+        switch (RobotMap.ROBOT_TYPE) {
+            case Frankenstein:
+            case pegaSwerve:
+                io = new RealGyro();
+                break;
+            case SIMULATION:
+                io = new SimGyro();
+                System.out.println("[Init] sim gyro");
```

Comment:
Don't we have logging, why print to the console?

Reply:
because i wanted to check if its initializing the right thing because at the beginning it didn't give output. eventually i found that the problem was at another part of the code